### ISSUE TITLE
Special Category Sugguesttions

### ISSUE THREAD

**Issue by NAFEDUDE** [Deinonychus71/Sm5shMusic#20]
I am unsure if we know how the game handles it, but the following aspects are impacted by music selection as well:

Ground Theme - SMB2 has a Pinch Theme, but only on certain Mario Stages
Certain Splatoon tracks make Moray Towers load the Splatfest (night) version of the stage
KOF Stadium picks characters to some extent based on music

**Comment by Deinonychus71** [comment]
Yep, this as well as a few other fields are either not fully implemented yet or are still pretty much in R&D, so there is room for improvement.
That said, in the meantime you can enable the advanced mode and change any value manually.

**Comment by Segtendo** [comment]
Speak of the devil. I did look in to what makes Moray Towers appear at night.

In the songs that trigger night, the `special_category` is set as `0x12111561a7`. I had someone test it and confirmed that it worked.

**Comment by Deinonychus71** [comment]
Thanks, I may poke you and some people around next week to try and figure out more of that stuff out, as I think it's a small enough change that it can be added along with other small fixes/enhancements

**Comment by Deinonychus71** [comment]
Do we have more information on these?
it looks like the moray towers is pretty straightforward, but what happens if the flag is set on another stage? is it ignored or does it crash?

**Comment by NAFEDUDE** [comment]
Should just ignore it, otherwise it would crash when selected for FD and BF for vanilla.
